tests/utils/inputparams/testparams.go

```diff
@@ -96,52 +101,65 @@ func GetEndPoint2() string {
 
 // GetSwarmManager1 returns swarm manager node IP from the configured swarm cluster
 func GetSwarmManager1() string {
-	return os.Getenv("MANAGER1")
+	manager1 := os.Getenv("MANAGER1")
+	if manager1 == "" {
+		log.Printf("Manager node not found. Need this to run swarm test.")
```

Comment:
@shaominchen I have removed the log.Printf from here and keeping the assert in the swarm test. I do not think we should change it to log.Fatal as in that case, whole test run will stop. And keeping assert in swarm test will only stop the swarm test and not the whole test run.